doc/cli/vast-import.md

```diff
@@ -18,6 +18,24 @@ The above command signals the running node to ingest (i.e., to archive and index
 for later export) all Suricata events from the Eve JSON file passed via standard
 input.
 
+The import command batches parsed events in table slices. To control the
+batching, the following options are available:
```

Comment:
```suggestion
The import command parses events into table slices or *batches*.
The following options control the batching:
```

Reply:
Looks like you went with the previous wording in your new commit?

Reply:
You are right, I overlooked this one apparently.